packages/es-client/tests/test-granule-upsert.js

```diff
@@ -268,13 +407,49 @@ test('upsertGranule does not update "failed" granule record to "running" status
     esClient,
     updates,
     index: esIndex,
+  }, false);
+  const updatedEsRecord = await t.context.esGranulesClient.get(granule.granuleId);
+  t.like(updatedEsRecord, updates);
```

Comment:
Do we want to be comparing against updates? And should we explicitly check the status on the updatedEsRecord?

Reply:
> Do we want to be comparing against updates? 

I think so - this test is checking that the update happens, so we expect the index would have a record that matches the update. 

> And should we explicitly check the status on the updatedEsRecord?

I think comparing it to the expected update object where it's specified is sufficient in this case, but I'm open to making more direct assertions if that's warranted.